lib/ansible/config/base.yml

```diff
@@ -1101,6 +1101,14 @@ DEFAULT_VAULT_IDENTITY:
   ini:
   - {key: vault_identity, section: defaults}
   yaml: {key: defaults.vault_identity}
+DEFAULT_VAULT_ENCRYPT_IDENTITY:
+  name: Vault id to use for encryption
+  default:
+  description: 'The vault of of the vault secret to use for encryption if not otherwise specified'
```

Comment:
"of of".
I think for new descriptions we should add more detail, clearly explaining when this is and is not needed, how it will be used, etc.